docs/sources/setup-grafana/configure-security/configure-authentication/google/index.md

```diff
@@ -253,3 +255,22 @@ In this example, all users will be assigned `Viewer` role regardless of the user
 role_attribute_path = "'Viewer'"
 skip_org_role_sync = false
 ```
+
+#### Org roles mapping examples
+
+##### Map organization roles
+
+Given a user member of the `example-group1@google.com` and `example-group2@google.com` Google groups.
```

Comment:
This is not a properly structured sentence.  Use something like "Let's say we have the following Google group:  `example-group1@google.com` and `example-group2@google.com`. "  Or "Here are 2 sample Google group": `example-group1@google.com` and `example-group2@google.com`.